frontsite/views.py

```diff
@@ -19,3 +23,36 @@ def index(request):
             'form': form,
             'job': job
         })
+
+def user_login(request):
+    template = 'login.html'
+
+    if request.method == 'POST':
+        # We use request.POST.get('<variable>') as opposed
+        # to request.POST['<variable>'], because the
+        # request.POST.get('<variable>') returns None if the
+        # value does not exist, while request.POST['<variable>']
+        # will raise a KeyError exception.
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+        # Check username/pw. Returns a User object or None
+        user = authenticate(username=username, password=password)
+        # If we have a User object, the details are correct.
+        if user:
+            if user.is_active:
+                login(request, user)
+                return render(request, template, {'message': "You are logged in successfully!"})
+            else:
+                return render(request, template, {'message': "Your account is disabled."})
+        else:
+            print("Invalid login details: {0}, {1}".format(username, password))
```

Comment:
Please remove `print` here. It's for debugging in development, not production environment.